HEC.FDA.Model/structures/Structure.cs

```diff
@@ -62,6 +69,33 @@ public Structure(string fid, PointM point, double firstFloorElevation, double va
             Notes = notes;
             Description = description;
         }
+        /// <summary>
+        /// Preferred Constructor for now. Eventually will store Geospatial.Vectors.Point instead of PointM.
+        /// </summary>
+        public Structure(string fid, Geospatial.Vectors.Point point, double firstFloorElevation, double val_struct, string st_damcat, string occtype, int impactAreaID, double val_cont = 0, double val_vehic = 0, double val_other = 0, string cbfips = "unassigned", double beginDamage = utilities.IntegerGlobalConstants.DEFAULT_MISSING_VALUE, double groundElevation = utilities.IntegerGlobalConstants.DEFAULT_MISSING_VALUE, double foundationHeight = utilities.IntegerGlobalConstants.DEFAULT_MISSING_VALUE, int year = utilities.IntegerGlobalConstants.DEFAULT_MISSING_VALUE, int numStructures = 1, string notes = "", string description = "")
+
+        {
+            Fid = fid;
+            PointM pt = Converter.ConvertPtM(point);
```

Comment:
Here we're bouncing back to PointMs. We're married to these for sampling from RAS6 things.  I think our best path forward is to always stay in 2025 land, because we have better access to fix problems directly through me, and through relationships with the team on that project, rather than 6.x, which isn't being maintained to the same degree in C#.